Tests/FisherYates/FisherYatesShuffleTest.swift

```diff
@@ -0,0 +1,33 @@
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+*/
+
+@testable import FisherYates
+import XCTest
+
+class FisherYatesShuffleTest: XCTestCase {
+    
+    func testShuffleInPlace(){
+        let hand1 = [3,4,5,6,7]
+        var hand2 = hand1
+        hand2.shuffleInPlace()
+        XCTAssertNotEqual(hand1, hand2)
```

Comment:
This test will fail ~1% of the time, it needs to be written in such a way that it always passes.